blocks/hero-carousel/hero-carousel.css

```diff
@@ -20,305 +21,356 @@
 
 .hero-carousel .carousel-slides {
   display: flex;
-  scroll-behavior: smooth;
   scroll-snap-type: x mandatory;
-  overflow: scroll clip;
+  overflow: auto;
+  scroll-behavior: smooth;
+  align-items: stretch;
+  scrollbar-width: none;
+  transition: transform 0.5s ease-in-out;
+}
+
+.hero-carousel .carousel-slide {
+  display: block;
+  position: relative;
+  flex-shrink: 0;
+  scroll-snap-align: start;
+  width: 100%;
+  height: 440px;
+  overflow: hidden;
 }
 
 .hero-carousel .carousel-slides::-webkit-scrollbar {
   display: none;
 }
 
 .hero-carousel .carousel-slide.background-light-green {
-  background-color: #E3F6E9;
+  background-image: url('https://cdn.piramalfinance.com/pchfweb/assets/images/pngs/WhatsappBannerMobile.png');
```

Comment:
the background-image could be used from DAM? (should be authorable via UE)

Reply:
as discussed, added all images to github and using relative paths in css.

kept the image format as-is, instead of converting to svgs.. let me know if there are guidelines or additional advantages in converting to svg always.